docs/capabilities.md

```diff
@@ -107,7 +107,9 @@ CAP_QUERY_TOP_PERCENT | Set to 'yes' if TOP supports percent | no | **<span styl
 CAP_QUERY_TOP_SAMPLE | Set to 'yes' if TOP supports sampling rows | no | **<span style="color:red">?</span>**  
 CAP_QUERY_TOP_SAMPLE_FAST | Set to 'yes' if TOP with sampling is faster than TOP | no | no 
 CAP_QUERY_TOP_SAMPLE_PERCENT | Set to 'yes' if TOP supports sampling by percentages | no | **<span style="color:red">?</span>**  
-CAP_QUERY_USE_DOMAIN_RANGES_OPTIMIZATION | Set to 'no' if domain ranges cannot be used to optimize set functions. | yes | yes
+CAP_QUERY_USE_DOMAIN_EXCEPT_OPTIMIZATION | When the number of selected items in a filter is greater than half the total number of items in the domain, save the excluded items instead of the included items to save space. | yes | yes 
```

Comment:
```suggestion
CAP_QUERY_USE_DOMAIN_EXCEPT_OPTIMIZATION | When the number of selected items in a filter is greater than half the total number of items in the domain, save the excluded items instead of the included items to save space.  Available in Tableau 2022.2 and newer. | yes | yes 
```